forms/copy.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+from wtforms.validators import Regexp
+from wtforms_alchemy import ModelForm
+from flask_wtf import FlaskForm
+
+from models import Copy
+
+asset_code_regex = Regexp('^[A-Z]{2}[0-9]{6}$',
+                          flags=re.IGNORECASE,
+                          message='Insert valid asset code.')
```

Comment:
I would add example of valid asset code here.
Or info what format of asset code is supported